tools/klee-stats/klee-stats

```diff
@@ -21,30 +21,47 @@ import collections
-    ('AvgMem(MB)', 'megabytes of memory currently used', "AvgMem"),
-    ('Queries', 'number of queries issued to STP', "NumQueries"),
-    ('AvgQC', 'average number of query constructs per query', "AvgQC"),
-    ('Tcex(s)', 'time spent in the counterexample caching code', "CexCacheTime"),
-    ('Tcex(%)', 'relative time spent in the counterexample caching code wrt wall time', "RelCexCacheTime"),
-    ('Tfork(s)', 'time spent forking', "ForkTime"),
-    ('Tfork(%)', 'relative time spent forking wrt wall time', "RelForkTime"),
+    ('TSolver(%)', 'time spent in the solver chain (incl. caches and constraint solver) wrt wall time', "RelSolverTime"),
+    # extended stats
+    # - code and coverage
+    ('ICovered', 'total covered instructions in the LLVM bitcode', "CoveredInstructions"),
+    ('IUncovered', 'total uncovered instructions in the LLVM bitcode', "UncoveredInstructions"),
+    ('Branches', 'number of conditional branch (br) instructions in the LLVM bitcode', 'NumBranches'),
+    ('FullBranches', 'number of fully-explored branch (br) instructions in the LLVM bitcode', 'FullBranches'),
+    ('PartialBranches', 'number of partially-explored conditional branch (br) instructions in the LLVM bitcode', 'PartialBranches'),
+    # - time
+    ('TUser(s)', 'total user time', "UserTime"),
     ('TResolve(s)', 'time spent in object resolution', "ResolveTime"),
     ('TResolve(%)', 'time spent in object resolution wrt wall time', "RelResolveTime"),
+    ('TCex(s)', 'time spent in the counterexample caching code', "CexCacheTime"),
+    ('TCex(%)', 'relative time spent in the counterexample caching code wrt wall time', "RelCexCacheTime"),
+    ('TQuery(s)', 'time spent in the constraint solver', "QueryTime"),
+    ('TSolver(s)', 'time spent in the solver chain (incl. caches and constraint solver)', "SolverTime"),
+    # - states
+    ('ActiveStates', 'number of currently active states (0 after successful termination)', "NumStates"),
+    ('MaxActiveStates', 'maximum number of active states', "MaxStates"),
+    ('AvgActiveStates', 'average number of active states', "AvgStates"),
+    # - constraint caching/solving
+    ('Queries', 'number of queries issued to the constraint solver', "NumQueries"),
+    ('QueryConstructs', 'number of query constructs for all queries send to the constraint solver', "NumQueryConstructs"),
+    ('AvgQC', 'average number of query constructs per query issued to the constraint solver', "AvgQC"),
     ('QCexCMisses', 'Counterexample cache misses', "QueryCexCacheMisses"),
     ('QCexCHits', 'Counterexample cache hits', "QueryCexCacheHits"),
+    # - memory
+    ('Mem(MiB)', 'mebibytes of memory currently used', "MallocUsage"),
+    ('MaxMem(MiB)', 'maximum memory usage (MiB)', "MaxMem"),
+    ('AvgMem(MiB)', 'average memory used (MiB)', "AvgMem"),
+    # - debugging
+    ('TArrayHash(s)', 'time spent hashing arrays (if KLEE_ARRAY_DEBUG enabled, otherwise -1)', "ArrayHashTime"),
```

Comment:
> otherwise -1
It won't be part of any DB entry in this case, so not shown at all

Reply:
the #ifdef is not in the code anymore: https://github.com/klee/klee/blob/e2abd854b8267dadd2399b70d1cda9a6acd6d737/lib/Core/StatsTracker.cpp#L463